### ISSUE TITLE
Getting started - what's the next step after installing client/server to connect LMS - i.e a working demo....

### ISSUE THREAD

**Issue by rodneytamblyn** [Cvmcosta/ltijs-demo-server#21]
Hi there,

I have successfully setup the client and server packages.  When I run the client package the UX appears.

My Mongo database is completely empty at this stage.

Can someone talk me through how to configure an LMS in the database.  

Assuming I am able to expose the test server on a temporary URL at mydomain.host and I wanted to connect this temporary app to an LMS, for example the free Canvas LMS service for teachers.  What values would I need to configure in the Mongo database (?) to get this sample app working.

Basically I am asking what is the next step after installing the client and server apps to a working demo please?

Thank you.

Rodney

**Comment by siddrcrelias** [comment]
I too am baffled at this point...like what after setting up the server and client...if I figure it out...I will update here for sure...

**Comment by Laktus** [comment]
@siddrcrelias Any updates? Struggling to understand how i can continue past setup. Don't even really understand how the client and server are connected.

**Comment by siddrcrelias** [comment]
@Laktus  ok I figured stuff out and made a article on it ...and managed to get a client live as well :)
https://medium.com/@debu2in/implementing-ltiv1-3-using-ltijs-1ab38ab87567

**Comment by siddrcrelias** [comment]
> @siddrcrelias Any updates? Struggling to understand how i can continue past setup. Don't even really understand how the client and server are connected.

so in the demo app by @Cvmcosta he has built the client and put it inside the public folder...and the ltijs library internally uses `ltijs-sequelize` which uses a database to store platform related information when you register the platform
basically you can build more functionality around the  `ltijs` library and extend the functionalities :) ... do reach out if you have any questions

**Comment by moomoo-dev** [comment]
Has anybody use this library to setup with Brightspace? 

I'm trying to find my OpenID Connect Login, but I'm 100% lost because I've never worked with this before. 

Thank you for your help
